lib/resolver.content.js

```diff
@@ -26,6 +26,10 @@ module.exports = class PodletClientContentResolver {
         Object.defineProperty(this, 'agent', {
             value: options.agent,
         });
+
+        this.metrics.on('error', error => {
+            this.log.error('Error emitted by metric stream', error);
```

Comment:
would it be helpful to add context info to this error log? eg. url to the podlet that was being resolved?